python/torch_mlir_e2e_test/test_suite/__init__.py

```diff
@@ -47,3 +47,4 @@ def register_all_tests():
     from . import return_types
     from . import control_flow
     from . import stats
+    from . import indirect_memory
```

Comment:
I think we can put these with the `index_put` tests. We can rename the file to `scatter`

Reply:
Refactored into `scatter.py`